images/macos/helpers/Common.Helpers.psm1

```diff
@@ -147,4 +147,23 @@ function Start-DownloadWithRetry
     }
 
     return $filePath
+}
+
+function Add-EnvironmentVariable
+{
+    param
+    (
+        [Parameter(Mandatory)] [string] $Name,
+        [Parameter(Mandatory)] [string] $Value,
+        [string] $FilePath = "${env:HOME}/.bashrc"
+    )
+
+    $envVar = "export {0}={1}" -f $name, $value
+    Tee-Object -InputObject $envVar -FilePath $filePath -Append
+}
+
+function Get-ToolcacheFromToolset {
```

Comment:
do we really need this additional function? We already have Get-ToolsetValue

Reply:
Removed `Get-ToolcacheFromToolset` function and used `Get-ToolsetValue`